internal/cli/watch_opts.go

```diff
@@ -68,22 +66,13 @@ func (opts *WatchOpts) Watch(f Watcher) error {
 }
 
 func (opts *WatchOpts) exponentialBackoff(err error) bool {
-	if opts.n <= 8 && checkForError(err, "CLUSTER_NOT_FOUND") {
-		backoff := math.Pow(base, float64(opts.n))
-		opts.n *= 2
-		sleepTime := time.Duration(backoff) * time.Second
-		time.Sleep(sleepTime)
-		return true
-	}
-	return false
-}
-
-func checkForError(err error, code string) bool {
+	backoffTimes := []time.Duration{1 * time.Second, 2 * time.Second, 4 * time.Second}
```

Comment:
[q] can this be made a `const`?

Reply:
It can't be a `const` as we're using a initializer and it's evaluated in a runtime, but `const` requires value that can be evaluated during the compile time.